dataactbroker/handlers/fileHandler.py

```diff
@@ -711,42 +704,18 @@ def upload_detached_file(self, create_credentials):
                 existing_submission = None
                 existing_submission_obj = None
 
-            for request_field, submission_field in request_job_mapping.items():
```

Comment:
Since for fabs submissions we're not getting reporting dates from the front end, this section of the code was removed which would handle the incoming reporting dates from the front end.